Week1/QA_PREP_EXERCISE.md

```diff
@@ -0,0 +1,17 @@
+As a preparation step for the upcoming Q&A, you will work through the following exercise. The expected result is a list of SQL `CREATE TABLE` statements that represent the tables you're being asked to design. We suggest you first think of the design of the tables (names, columns, types) and then proceed to write the SQL statements, that way you are thinking about the problem you're trying to solve (creating a database for a website), rather than the implementation first (creating a set of SQL statements).
+
+### Option 1
```

Comment:
I would simply call it `Exercise 1` instead of `Option 1`. Some students might interpret it as they can do one _or_ the other.